src/planner/planners/MatchSolver.h

```diff
@@ -22,6 +25,21 @@ class MatchSolver final {
     static Expression* rewrite(const LabelExpression *label);
 
     static Expression* rewrite(const LabelAttributeExpression *la);
+
+    static Expression* doRewrite(const MatchAstContext* mctx, const Expression* expr);
+
+    static bool match(AstContext* astCtx);
+
+    static Expression* makeIndexFilter(const std::string& label,
+                                       const MapExpression* map,
+                                       QueryContext* qctx);
+
+    static Expression* makeIndexFilter(const std::string& label,
+                                       const std::string& alias,
+                                       Expression* filter,
+                                       QueryContext* qctx);
+
+    static Status buildFilter(const MatchAstContext* mctx, SubPlan* plan);
```

Comment:
Don't mix use `build` and `make`.

Reply:
Only move existing codes here, well, I will fix it in next zero to one step PR.